deploy-cloud-functions/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "deploy-cloud-functions",
+  "version": "0.1.0",
+  "description": "Deploy a Cloud Function",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "ncc build src/main.ts",
+    "lint": "eslint . --ext .ts,.tsx",
+    "fixlint": "eslint . --ext .ts,.tsx --fix",
+    "format": "prettier --write **/*.ts",
+    "test": "mocha -r ts-node/register -t 600s 'tests/*.test.ts'",
+    "e2e-tests": "mocha -r ts-node/register -t 120s 'tests/e2e/e2e.test.ts'",
+    "cleanup": "mocha -r ts-node/register -t 240s 'tests/cleaner/cleanTestFunctions.test.ts'"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run build && git add dist/"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/github-actions"
+  },
+  "keywords": [
+    "actions",
+    "gcf",
+    "functions",
+    "google cloud",
+    "cloud function"
+  ],
+  "author": "GoogleCloudPlatform",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@actions/core": "^1.2.4",
+    "@types/archiver": "^3.1.0",
+    "archiver": "^5.0.0",
+    "fs": "0.0.1-security",
+    "gaxios": "^3.1.0",
+    "googleapis": "^58.0.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.12",
+    "@types/lodash": "^4.14.159",
+    "@types/mocha": "^8.0.1",
+    "@types/node": "^14.0.27",
+    "@types/uuid": "^8.0.1",
+    "@typescript-eslint/eslint-plugin": "^3.8.0",
+    "@typescript-eslint/parser": "^3.8.0",
+    "@zeit/ncc": "^0.22.3",
+    "chai": "^4.2.0",
+    "eslint": "^7.6.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "google-auth-library": "^6.0.6",
```

Comment:
Should this be in dependencies not devDependencies?

Reply:
Good catch. I was wondering if its because we do `npm install` by default and install everything before build.
Also noticed it here https://github.com/GoogleCloudPlatform/github-actions/blob/8b66b2e3e755acd3aac3179ea6a38ac4789db8a2/deploy-cloudrun/package.json#L50

Reply:
Updated on my PR as well

Reply:
Depending on what's needed - we should update to `npm i --only=prod`